requirements.txt

```diff
@@ -0,0 +1,33 @@
+attrs==18.1.0
```

Comment:
I think we don't need all of these requirements. Please add only required ones.

Reply:
Sorry, I auto-generated that.